docs/documentation/release_notes/topics/26_1_0.adoc

```diff
@@ -108,6 +108,22 @@ The key providers that allow to import externally generated keys (`rsa` and `jav
 
 The default `generated` key providers generate a certificate valid for 10 years (the types that have or can have an associated certificate). Because of the long validity and the recommendation to rotate keys frequently, the generated providers do not perform this check.
 
+= OpenTelemetry Tracing supported
+
+In the previous release, the OpenTelemetry Tracing feature was preview and is *fully supported* now.
+It means the `opentelemetry` feature is enabled by default.
+
+There were made multiple improvements to the tracing capabilities in {project_name} such as:
+
+* Introduced private (for now) *Tracing SPI* for instrumenting {project_name} codebase
```

Comment:
IMHO we should not be announcing private SPIs in release notes as it might set the wrong expectations in terms of support.

Reply:
TL;DR I thought about it as well, but put it there anyway.

The thing is that I'd like to provide some tutorial/guide on how users can instrument their own Keycloak extensions to be able to use traces. I'm not sure what are Keycloak policies around using private SPIs, but AFAIK there is no support for them and they can be changed (or even removed?). Anyway, I'd like to promote somehow that they can use traces in their extensions, and mentioning it in the release notes seems to me like a good start to be aware of such functionality. That's also why I include the "for now" to emphasize it's gonna be there for some time. 

@vmuzikar I don't think it's such a big deal to keep it there, but if some of you think otherwise, we can set the proper way :))

Reply:
If we actually want to promote it and let folks use it, IMHO we should've made the SPI public.

But in the current state when it's private, I'd really remove it. It gives folks mixed message ("it's promoted in the release notes but it's marked as private – not even a tech preview – so I shouldn't use it?")

Reply:
Ok, I'll remove it and then we can discuss more the support for the SPI.

Reply:
@vmuzikar I removed the note, but kept the sub-list for custom spans as there's gonna be another note about Keycloak CR configuration.